source/_integrations/spotify.markdown

```diff
@@ -17,6 +17,8 @@ ha_platforms:
 
 The Spotify media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
 
+Once an output [source]](#selecting-output-source) is selected you will be able to play content from Spotify using the standard Media Browser UI or `media_player` service.
```

Comment:
I don't think this belongs in the introduction

Reply:
You are right the source detail doesn't belong here. As a new user I didn't even realise where to look next, so how about links?

I will push what I mean